docs/user-guide/auto_sync.md

```diff
@@ -18,6 +18,16 @@ spec:
   syncPolicy:
     automated: {}
 ```
+Application CRD now also support explicitly setting automated sync to be turn on or off  by using `spec.syncPolicy.automated.enable` flag to true or false
+```yaml
+spec:
+  syncPolicy:
+    automated:
+      enable: true
+```
+
+!!!note 
+  **Note:** Setting `spec.syncPolicy.automated.enable` flag to null will be treated as automated sync as enabled.
```

Comment:
Since you're using a note admonition widget already, which is titled "Note", it feels weird to have another title of "Note" there.

I think the behavior of the `enabled` field and the reason it exists should be a little better explained here.

Reply:
Thanks for the review
I have updated the description and note, can you check if the updated documentation explains the behaviour better?